lib/workers/global/index.js

```diff
@@ -50,5 +50,6 @@ function getRepositoryConfig(globalConfig, index) {
   });
   repoConfig.isGitHub = repoConfig.platform === 'github';
   repoConfig.isGitLab = repoConfig.platform === 'gitlab';
+  repoConfig.isGitvsts = repoConfig.platform === 'vsts';
```

Comment:
Perhaps this can just be `isVsts` ? i.e. not sure if we need to prefix vsts with git?

Reply:
On this one, yes I would like to keep isGitvsts since vsts supports also SVN and another one that I can't remember the name